src/devtools/views/components/ComponentInspector.vue

```diff
@@ -84,4 +110,20 @@ export default {
 
 .component-name
   margin 0 10px
+
+.data
+  padding: 20px 0px
+
+.data-container
+  float: left
+  min-width: 33%
+  padding: 0px 10px
+
```

Comment:
I'd move the `float` and `min-width` for `.data-container` into a media query here so it stacks when we don't have the horizontal space. I'd *try* this. I didn't test it at all but should get you close.

```stylus
@media (min-width: $wide)
  float left
  width 33.333%
  overflow overlay
  ```